### ISSUE TITLE
404 error when trying to run json-server

### ISSUE THREAD

**Issue by Luckygirlllll** [typicode/json-server#1201]
I'm trying to run json-server, and I'm getting 404 error, I followed all instructions, however, it still doesn't work. 

I'm using db, which is in the example. 
What can be wrong?

Here are screenshots:
<img width="559" alt="Screenshot 2020-11-26 at 17 29 42" src="https://user-images.githubusercontent.com/10096433/100374795-1a5de000-300d-11eb-9c98-14308e9908c6.png">

<img width="660" alt="Screenshot 2020-11-26 at 17 29 54" src="https://user-images.githubusercontent.com/10096433/100374823-221d8480-300d-11eb-9cd4-f67bdcf18ce1.png">

**Comment by RaduGrama** [comment]
@Luckygirlllll I experienced the same issue and I noticed that port 3000 was already in use by another API - that may or may not be the case for you. At any rate, if the port is already in use, there should be some sort of a failure/message.

**Comment by Luckygirlllll** [comment]
@RaduGrama true, in my case port 3000 also was in use, when I changed port to another one it started work!
